### PR TITLE
Add template version to AndroidManifest.xml and Info.plist

### PR DESCRIPTION
**Pull request by filiph**
For tracking purposes, we're adding a key-value pair to each game template. This can be used by the AdMob plugin to recognize that this is a Flutter game built on top of the template.

## Pre-launch Checklist

- [x] I read the [Flutter Style Guide] _recently_, and have followed its advice.
- [x] I signed the [CLA].
- [x] I read the [Contributors Guide].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-devrel channel on [Discord].

[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[CLA]: https://cla.developers.google.com/
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[Contributors Guide]: https://github.com/flutter/games/blob/main/CONTRIBUTING.md

### CODE REVIEW

**Review by domesticmouse — APPROVED** [review]
Is template version all you want to track? Also, you may want to add a comment justifying to the template user why they should leave it in their published game.

File: templates/basic/android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -19,6 +19,10 @@
               android:name="io.flutter.embedding.android.NormalTheme"
               android:resource="@style/NormalTheme"
               />
+            <!-- Specifies the version of the game template for tracking purposes. -->
```
Comment (domesticmouse) [inline templates/basic/android/app/src/main/AndroidManifest.xml:22]: Change to "Specifies the version of the game template to understand template usage."

File: templates/card/android/app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -19,6 +19,10 @@
               android:name="io.flutter.embedding.android.NormalTheme"
               android:resource="@style/NormalTheme"
               />
+            <!-- Specifies the version of the game template for tracking purposes. -->
```
Comment (domesticmouse) [inline templates/card/android/app/src/main/AndroidManifest.xml:22]: Change to "Specifies the version of the game template to understand template usage."

File: templates/endless_runner/android/app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -19,6 +19,10 @@
               android:name="io.flutter.embedding.android.NormalTheme"
               android:resource="@style/NormalTheme"
               />
+            <!-- Specifies the version of the game template for tracking purposes. -->
```
Comment (domesticmouse) [inline templates/endless_runner/android/app/src/main/AndroidManifest.xml:22]: Change to "Specifies the version of the game template to understand template usage."

**Review by domesticmouse — CHANGES_REQUESTED** [review]

**Comment by filiph** [comment]
I've changed the wording.

**Review by domesticmouse — APPROVED** [review]

**Pull request merged**